spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -526,21 +485,22 @@ public class TransformProcessor {
   }
 }
 ----
-====
 
-===== Using @StreamListener for Dispatching Messages to Multiple Methods
 
-Since version 1.2, Spring Cloud Stream supports dispatching messages to multiple `@StreamListener` methods registered on an input channel, based on a condition.
+==== Using @StreamListener for content-based routing (route to multiple handler methods)
```

Comment:
Replace with "==== Using `@StreamListener` for Content-based Routing (Routing to Multiple Handler Methods)"

(Awfully long heading - any way to make two sections or shorten the heading? If not, we can live with it.)